HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -86,6 +165,18 @@ def customiseFor18429(process):
 
      return process
 
+# Add new parameters required by RecoTauBuilderConePlugin
+def customiseFor18429(process):
```

Comment:
Hi @afiqaize - seems a duplicate definition of customiseFor18429 has been picked up. Please remove.